### PR TITLE
[MS] Updated contract page

### LINKED ISSUE #7967: [Client - Sellsy] Contract page
https://www.figma.com/design/8WXUivEfT1x4FTzCr3WNxd/Parsec-Client?node-id=200-7029&t=M6Eot8DLK14rZkZo-11

link to #7984

### PR DESCRIPTION
**Pull request by fabienSvtr**
## Describe your changes

image

## Checklist

Before you submit this pull request, please make sure to:

- [ ] Keep changes in the pull request as small as possible
   
- [ ] Ensure the commit history is sanitized
   
- [ ] Give a meaningful title to your PR
- [ ] Describe your changes
   
- [ ] Link any related issue in the description
   
- [ ] Link any dependent pull request in the description

closes #7967

### CODE REVIEW

**Review by Max-7 — CHANGES_REQUESTED** [review]
Blocking to make sure `bms/api.ts` gets the right default values back.

File: client/src/locales/en-US.json [hunk 1]
```diff
@@ -1455,26 +1485,35 @@
             "title": "Billing address"
         },
         "contracts": {
+            "loading": "Your contrat arrives",
```
Comment (NicoTuxx) [inline client/src/locales/en-US.json:1488]:
```suggestion
            "loading": "Your contract arrives",
```

File: client/src/locales/en-US.json [hunk 2]
```diff
@@ -1362,6 +1362,36 @@
...
+                "invoicePaid": "Active",
+                "invoiceToBePaid": "Invoice to be paid",
+                "nothingLinked": "Being processed"
+            },
+            "number": "Invoice number",
+            "information": "Information",
+            "startingDate": "Starting date",
+            "endingDate": "Ending date",
+            "members": {
+                "title": "Licence member | Licences members",
```
Comment (NicoTuxx) [inline client/src/locales/en-US.json:1380]:
```suggestion
                "title": "License member | Licenses members",
```

File: client/src/locales/en-US.json [hunk 3]
```diff
@@ -1362,6 +1362,36 @@
...
+                "admin": "Administrator member | Administrator members",
+                "adminDescription": "Full access",
+                "standard": "Standard member | Standard members",
+                "standardDescription": "Classic use",
+                "external": "External member | External members",
+                "externalDescription": "Limited use"
+            },
+            "storage": {
+                "title": "Licence storage",
```
Comment (NicoTuxx) [inline client/src/locales/en-US.json:1389]:
```suggestion
                "title": "License storage",
```

**Review by NicoTuxx — CHANGES_REQUESTED** [review]

**Review by NicoTuxx — APPROVED** [review]

**Review by Max-7 — APPROVED** [review]

**Pull request merged**